modules/storage/local.go

```diff
@@ -128,8 +128,13 @@ func (l *LocalStorage) URL(path, name string) (*url.URL, error) {
 }
 
 // IterateObjects iterates across the objects in the local storage
-func (l *LocalStorage) IterateObjects(fn func(path string, obj Object) error) error {
-	return filepath.WalkDir(l.dir, func(path string, d os.DirEntry, err error) error {
+func (l *LocalStorage) IterateObjects(prefix string, fn func(path string, obj Object) error) error {
+	dir := l.dir
+	prefix = strings.TrimSpace(prefix)
```

Comment:
It's a friendly correction, however, what if I really want a file with path `/a/b /c.txt`? It's fine for both filesystem and object storage.

Reply:
Yes. blank space is correct in filesystem. no need to trim.